packages/owl-bot/test/fake-configs-store.ts

```diff
@@ -80,6 +80,11 @@ export class FakeConfigsStore implements ConfigsStore {
     }
   }
 
+  clearConfigs(repo: string): Promise<void> {
+    this.configs.delete(repo);
+    return Promise.resolve();
+  }
```

Comment:
Minor: you can add the `async` keyword which always returns a promise (even if the contents are synchronous)

```suggestion
  async clearConfigs(repo: string): Promise<void> {
    this.configs.delete(repo);
  }
```